docs/source/install/path.rst

```diff
@@ -114,9 +114,16 @@ References
 .. [Spack]
         T. Gamblin and contributors.
         *A flexible package manager that supports multiple versions, configurations, platforms, and compilers*,
+        SC '15 Proceedings of the International Conference for High Performance Computing, Networking, Storage and Analysis (2015),
```

Comment:
They don't do this quite consequently, I used the spelling from [here](https://dx.doi.org/10.1145/2807591.2807623)